Tech-Matrix/PRNTRS
# Payment Gateway for Client 2/2 : Implementing the Frontend for the Stripe Payment Integration

#### Issue Number
fixes #26 

#### Describe the changes you've made
- Fixed the route to `mappy` in `index.js`
- Added a button and other required code to integrate Payment with Frontend `mappy` page.
- For the routes to get access to our HTML and CSS files I have added the `ejs` templates in `views` and static files in `public`

#### Describe if there is any unusual behaviour of your code ( Write `NA` if there isn't )
The CSS for the `mappy` page and Landing page after redirection does not load. This can be fixed in another PR/new issue.

#### Additional context

**The PR has 473 additions but the only crucial one is the index.js file and the mappy html changes**
#### Test plan ( Optional but Recommended )
- Get Stripe API keys from the website.
- Fork, fetch and run the `index.js` file
- Click the Payment link on `mappy.html` to initiate payment
- Use `4242 4242 4242 4242` as test card number, give any future date as expiry and any 3 digit CVV
- You should be redirected to Landing Page.

#### Checklist
